lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -778,17 +761,60 @@ def get_file(file, use_token: true)
         test_output_against_profile('Procedure', profile_definitions)
       end
 
-      test :validate_medication do
+      test :validate_encounter do
+        metadata do
+          id '18'
+          name 'Encounter resources on the FHIR server follow the US Core Implementation Guide'
```

Comment:
```suggestion
          name 'Encounter resources returned conform to the US Core Encounter Profile.'
```

Reply:
Changed to US Core Implementation Guide xxx Profile(s) for all tests